src/services/technicalRecord.service.ts

```diff
@@ -0,0 +1,65 @@
+import { NewPlateRequest } from '../models/Request.model';
+import { Plates } from '../models/Plates.model';
+import { getSerialNumber } from './serialNumber.service';
+import { TechRecord, Vehicle } from '../models/Vehicle.model';
+import { put } from './dynamodb.service';
+import logger from '../observability/logger';
+
+export const addNewPlate = async (request: NewPlateRequest): Promise<TechRecord[]> => {
+  if (!request.reasonForCreation || !request.vtmUsername || !request.techRecord) {
+    throw new Error('Bad Request');
+  }
+
+  const techRecords = request.techRecord as TechRecord[];
+
+  // find the current record then add the plate to it
+  const currentTechRecordIndex = techRecords.findIndex((techRecord) => techRecord.statusCode === 'current');
+
+  if (currentTechRecordIndex < 0) {
+    throw new Error('No current record found');
+  }
+
+  const currentPlates = techRecords[currentTechRecordIndex].plates ?? [];
+
+  const newPlate: Plates = {
+    plateSerialNumber: await getSerialNumber(),
+    plateIssueDate: new Date().toISOString(),
```

Comment:
This date appears as is on the plate PDF - and it's not human readable.
Do we make the date readable here or re-format lower down the flow?

Reply:
We are currently fixing cvs-svc-doc-gen to do this, this needs to stay as is for dynamo stuff :)